stdlib/public/stubs/LibcShims.cpp

```diff
@@ -37,6 +37,8 @@
 #include <string.h>
 #if __has_include(<sys/random.h>)
 #include <sys/random.h>
+#elif defined(__NR_getrandom)
+#include <sys/syscall.h>
```

Comment:
This could use a separate `__has_include` to avoid depending on whether `<unistd.h>` is included before or after.

```diff
 #include <stdlib.h>
 #include <string.h>
 #if __has_include(<sys/random.h>)
 #include <sys/random.h>
 #endif
 #include <sys/stat.h>
+#if __has_include(<sys/syscall.h>)
+#include <sys/syscall.h>
+#endif
 #include <sys/types.h>
 #include <type_traits>
```